src/pages/Trade/Trade.tsx

```diff
@@ -24,15 +21,6 @@ export const Trade = () => {
   } = useWallet()
   const top = isDemoWallet ? '7rem' : '4.5rem'
   const borderColor = useColorModeValue('gray.100', 'gray.750')
-  const headerBg = useColorModeValue('gray.100', 'gray.800')
```

Comment:
@gomesalexandre FYI this was the cause of the bug on the trade confirmated broadcast**ed** step - the new txid was causing the swapper to rerender and set the default assets back to ETH/FOX